azure-quantum/azure/quantum/target/quantinuum.py

```diff
@@ -72,14 +65,19 @@ def submit(
         :return: Azure Quantum job
```

Comment:
NIT: Update comment on line 55 since it can now accept qir

Reply:
When talking to Cesar, we decided to minimize the exposure of this functionality everywhere.